Client/Ecosia/UI/SemanticColor.swift

```diff
@@ -34,6 +34,10 @@ extension UIColor {
             static let primary = UIColor(red: 0.059, green: 0.059, blue: 0.059, alpha: 1)
         }
         
+        struct Brand {
+            static let primary = UIColor(red: 0, green: 0.5, blue: 0.033, alpha: 1)
```

Comment:
I also thought of adding this. Probably good to have it separately, though Button.primary is the same color 🎨